poll/poll.py

```diff
@@ -159,9 +159,9 @@ def gather_items(data, result, noun, field, image=True):
             else:
                 items.append([key, label])
 
-        if not len(items) > 1:
+        if not items:
```

Comment:
@e-kolpakov Breaking this method up into several methods for validation would be a bit outside the scope of this task, I feel, though I understand the principle. I'll keep it as is. If we end up changing validation significantly some day, we may have to do that though, yes.